infra/modules/container-app-environment.bicep

```diff
@@ -8,7 +8,8 @@ param workloadProfileType string
 param infrastructureSubnetId string
 param appInsightsConnectionString string
 
-var workloadProfileName = 'default'
+// var workloadProfileName = 'default'
+var workloadProfileName = 'Consumption'
```

Comment:
so you got ElasticSearch to run with much less memory?

Reply:
yeah, I can see elastic running, indexes get created and job is successful but the grafana job is failing (also locally)

everything works when I import the dashboard manually though :/